<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTabWidget" name="tap_info">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>PLC</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>0</y>
          <width>155</width>
          <height>28</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QToolButton" name="btn_addPlc">
           <property name="text">
            <string>신규</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="btn_delPlc">
           <property name="text">
            <string>삭제</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="btn_savePlc">
           <property name="text">
            <string>저장</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>30</y>
          <width>271</width>
          <height>461</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>패키지 명</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="PLC_PKG_ID">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>PLC 아이디</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="PLC_PLC_ID"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>MAKER</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="PLC_PLC_MAKER"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>CPU Type</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="PLC_CPU_TY"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Slot</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="PLC_SLOT"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Rack</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="PLC_RACK"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>통신타입</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="PLC_COMM_TY"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>아이피</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="PLC_PLC_IP"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>포트</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="PLC_PLC_PORT"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>로그 출력</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QComboBox" name="PLC_LOG_YN"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="PLC_USE_YN"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>사용여부</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="list_sk">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>801</width>
          <height>451</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <column>
         <property name="text">
          <string>새 행</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>53</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>PKG</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="plc_search_pkg">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>PLC id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="plc_search_plc">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>READ address</string>
       </attribute>
       <widget class="QLabel" name="selected_msg">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>20</y>
          <width>281</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>53</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>PKG</string>
        </property>
       </widget>
       <widget class="QWidget" name="formLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>30</y>
          <width>271</width>
          <height>461</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>패키지 명</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="READ_PKG_ID">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>PLC 아이디</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="READ_PLC_ID"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>사용여부</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="READ_USE_YN"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="READ_ADDR"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Pos</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="READ_POS"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Length</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="READ_LENGTH"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Alias</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="READ_ALIAS"/>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="addr_search_pkg">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>PLC id</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="list_sk_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>801</width>
          <height>451</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <column>
         <property name="text">
          <string>새 행</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="addr_search_plc">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>0</y>
          <width>155</width>
          <height>28</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QToolButton" name="btn_addAddr">
           <property name="text">
            <string>신규</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="btn_delAddr">
           <property name="text">
            <string>삭제</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="btn_saveAddr">
           <property name="text">
            <string>저장</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
